### ISSUE TITLE
[Events] Component to render all the details of a particular event

### ISSUE THREAD

**Issue by tusharjain0022** [houseofgeeks/hg#203]
### Description
<write your description> 
We require a component that will be visited when we click on the name of any event from the events table and take event details of that particular event as prop and render its details with full description on a separate page.

### Screenshots/testimonials
<place your screenshots or testimonials here if applicable>
NA

**Comment by tusharjain0022** [comment]
@EmperorYP7 , check the description and if you want to add or edit something to it.

**Comment by EmperorYP7** [comment]
We must also provide a rough wireframe/design for someone to get started with this. Although, we need to configure a bucket for storing banners and possibly a gallery later on. What do you think @ankiiitraj @tusharjain0022?

**Comment by ankiiitraj** [comment]
Agreed, but I think this will be overkill for such a small thing like an event! my opinion! your thoughts?

**Comment by EmperorYP7** [comment]
I was thinking the same. Configuring the bucket should be the part of v2.0 of this website. (We'll possibly have the next cohort working on this)

**Comment by SrijanSriv** [comment]
I fidgeted through the ways of showing the details of an event, but it really isn't aesthetically fitting. The description is really small(as expected). I suggest that we modify the `/events` tab where the events can have their description right below them. Unless anyone has something else in mind? Paging @tusharjain0022 @EmperorYP7 .

What I could think of was just this bit already present in the admin console, with slight modifications.
![image](https://user-images.githubusercontent.com/79690889/150330160-f03da9a9-4fdd-46b8-9cd9-a56b7d108bf1.png)
